Core/Interfaces/ICommentService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MovieFiles.Core.Models;
+
+namespace MovieFiles.Core.Interfaces
+{
+    public interface ICommentService
+    {
+        Task<List<Comment>> GetComments(int movieId, int page);
```

Comment:
Consider to return object that will include information about total number of pages